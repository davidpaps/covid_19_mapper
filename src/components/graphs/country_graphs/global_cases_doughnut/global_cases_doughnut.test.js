import * as React from "react";
import { shallow } from "enzyme";
import GlobalCasesDoughnut from "./global_cases_doughnut";

describe("World Doughnut", () => {
  let wrapper;

  beforeEach(
    () =>
      (wrapper = shallow(
        <GlobalCasesDoughnut country={""} countries={[]} total={[]} />
      ))
  );

  it("should render a <Doughnut />", () => {
    expect(wrapper.find("Doughnut").length).toEqual(1);
  });

  it("renders correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
