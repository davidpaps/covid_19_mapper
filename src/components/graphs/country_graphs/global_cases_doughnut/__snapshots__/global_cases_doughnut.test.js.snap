// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`World Doughnut renders correctly 1`] = `ShallowWrapper {}`;
